modules/react/select/lib/Select.tsx

```diff
@@ -94,6 +94,7 @@ export const SelectInput = createSubcomponent(TextInput)({
           disabled={disabled}
           placeholder={placeholder}
           error={error}
+          onFocus={onFocus}
```

Comment:
Can we also add ref support for this input as well? We also need to be able to get a ref to this element (not the hidden one)

Reply:
Unless the hidden input is inert, I _think_ this will double-fire the focus event.
How did the previous Select work around this problem?

Reply:
Just tested this. The hidden input is inert, so the event only fires once.